### PR TITLE
Upgrade to ember 2.13

### PR DESCRIPTION
**Pull request by cherewaty**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,46 +1,48 @@
...
-  - sh -e /etc/init.d/xvfb start
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin:$PATH
 
 install:
-  - npm install
-  - bower install
+  - yarn install --no-lockfile
```
Comment (Robdel12) [inline .travis.yml]: 🔥

File: package.json [hunk 2]
```diff
@@ -1,11 +1,22 @@
 {
   "name": "emberx-form",
-  "version": "0.0.4",
-  "description": "A set of components for building dynamic forms. Built on top of ember-changeset.",
+  "version": "0.0.0",
+  "description": "The default blueprint for ember-cli addons.",
```
Comment (Robdel12) [inline package.json]: Looks like the version & description were removed here

Comment (cherewaty) [inline package.json]: Thanks for the catch @Robdel12

Comment (Robdel12) [inline package.json]: _high five!_

**Pull request merged**